src/components/landing/header/Header.js

```diff
@@ -26,15 +26,29 @@ const Header = () => {
             guía y el apoyo de la comunidad lo lograrás!
           </p>
         </Fade>
-        <Fade bottom>
-          <Link href="/users/sign_in">
-            <Button
-              label="Quiero registrarme en Ser.dev"
-              disabled={currentUser}
-              className={`${currentUser ? 'hidden' : ''}`}
-            />
-          </Link>
-        </Fade>
+        <div className="flex gap-4">
+          <Fade bottom>
+            <Link href="/curriculums/1">
+              <Button
+                label={`${
+                  currentUser
+                    ? 'Quiero seguir estudiando'
```

Comment:
when the user is logged in, I think this button should be styled as a main button (blue background and white letters). It's the main course of action we want our clients to follow